Makefile

```diff
@@ -0,0 +1,45 @@
+TEST=tests
+PDB_PATH=$(shell find /usr/lib{,64} -type f -executable -name pdb.py 2> /dev/null)
+DATA=$(shell echo $(MODULE) | sed 's/\(foreman\|katello\)_//; s/_/-/g')
+MODULE_PATH=modules/$(MODULE).py
+DEBUG_DATA_PATH=tests/data/$(DATA).json
+DEBUG_OPTIONS=-m $(MODULE_PATH) -a @$(DEBUG_DATA_PATH) -D $(PDB_PATH)
```

Comment:
Sorry, to nag again: All these variables seem like perfect candidates for 'simply expanded variables', (`::=`) especially the ones calling shell commands.
See: https://www.gnu.org/software/make/manual/html_node/Flavors.html

Reply:
Why would that be better? Using recursively expanded variables, the variables are not even executed unless they are used (i.e. lazy assignment). That seems like the best case here; you wouldn't want `MODULE`, `DATA`, etc. to be getting set if you are just running `make setup`.

Reply:
Just to be sure: Recursively expanded variables are more like macros (evaluated on every use), not like lazy assigned statements (evaluated on first use only and not available im Make). So at least for calling shell commands i would prefer simple (immediate) expansion, to prevent multiple execution and possibly very strange side effects where the second execution evaluates differently. Maybe for a Makefile that size the whole discussion is moot. In the end it might be a matter of taste.